docs/reference/migration/migrate_7_0/search.asciidoc

```diff
@@ -43,7 +43,8 @@ Scroll queries are not meant to be cached.
 
 ==== Term Suggesters supported distance algorithms
 
-The following suggestion distance algorithms have been removed:
+For better readability and consistency the following string distance algorithms
```

Comment:
How about:

`The following string distance algorithms were renamed in 6.0 and their old names were deprecated. The deprecated names have now been removed.`

Reply:
How about:

>The following string distance algorithms were given additional names in 6.2 and their existing names were deprecated. The deprecated names have now been removed.

Reply:
Thanks for the suggestions!